From 28f48600ab4f1592146ca51fe5b0919da1b6523a Mon Sep 17 00:00:00 2001
From: David Trasbo <me@dtrasbo.com>
Date: Mon, 20 Sep 2010 13:12:10 +0200
Subject: [PATCH] Ruby 1.9 compatibility: Make response body an array in RoutingTest.

Rack calls #each on response bodies, but strings don't respond to that in 1.9.

[#5667 state:committed]
---
 railties/test/application/routing_test.rb |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/railties/test/application/routing_test.rb b/railties/test/application/routing_test.rb
index 42ee4c5..41f5da0 100644
--- a/railties/test/application/routing_test.rb
+++ b/railties/test/application/routing_test.rb
@@ -186,7 +186,7 @@ module ApplicationTests
 
       add_to_config <<-R
         routes.append do
-          match '/win' => lambda { |e| [200, {'Content-Type'=>'text/plain'}, 'WIN'] }
+          match '/win' => lambda { |e| [200, {'Content-Type'=>'text/plain'}, ['WIN']] }
         end
       R
 
-- 
1.7.2.3

